app/views/google-map/marker.js

```diff
@@ -47,6 +47,10 @@ export default GoogleMapCoreView.extend({
     return this.get('controller.infoWindowTemplateName') || this.get('parentView.markerInfoWindowTemplateName');
   }).readOnly(),
 
+  infoWindowViewClass: computed('controller.infoWindowViewClass', 'parentView.infoWindowViewClass', function () {
+    return this.get('controller.infoWindowViewClass') || this.get('parentView.infoWindowViewClass');
+  }).readOnly(),
```

Comment:
As explained up, `parentView.infoWindowViewClass` has to be renamed into `parentView.markerInfoWindowViewClass`